pkg/parser/quoted_text_test.go

```diff
@@ -77,6 +77,52 @@ var _ = Describe("quoted texts", func() {
 				Expect(ParseDraftDocument(source)).To(MatchDraftDocument(expected))
 			})
 
+			It("bold text with newline", func() {
+				source := "*some bold\ncontent*"
+				expected := types.DraftDocument{
+					Blocks: []interface{}{
+						types.Paragraph{
+							Lines: [][]interface{}{
+								{
+									types.QuotedText{
+										Kind: types.Bold,
+										Elements: []interface{}{
+											types.StringElement{Content: "some bold\ncontent"},
```

Comment:
now I realize that the `line` may actually span on multiple lines, but it's not a big deal I guess...